sdk/communication/azure-communication-rooms/src/test/java/com/azure/communication/rooms/RoomsTestBase.java

```diff
@@ -31,7 +31,7 @@
 import com.azure.core.http.rest.Response;
 import static org.junit.jupiter.api.Assertions.*;
 
-public class RoomsTestBase extends TestBase {
```

Comment:
I do see some redactions that need to be added. 
Currently, adding redactors directly to the record policy is not supported.
To add sanitizers follow - https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/core/azure-core-test#sanitize-secrets
needed below code changes
```
1) .httpClient(interceptorManager.isPlaybackMode() ? interceptorManager.getPlaybackClient() : httpClient)

2) builder.addPolicy(interceptorManager.getRecordPolicy());

3) FakeCredentials -> MockTokenCredential
````